src/services/db/__tests__/GitHubService.spec.ts

```diff
@@ -912,4 +913,39 @@ describe("Github Service", () => {
       })
     })
   })
+
+  describe("changeRepoPrivacy", () => {
+    const refEndpoint = `${siteName}`
+    const headers = {
+      Authorization: "",
+      "Content-Type": "application/json",
+    }
+    it("should modify the repo privacy accordingly if changing to true", async () => {
+      const shouldBePrivate = true
+      mockAxiosInstance.patch.mockResolvedValueOnce("")
+      const resp = await service.changeRepoPrivacy(sessionData, shouldBePrivate)
+      expect(mockAxiosInstance.patch).toHaveBeenCalledWith(
+        refEndpoint,
+        { private: shouldBePrivate },
+        {
+          headers,
+        }
+      )
+      expect(resp.isOk()).toEqual(true)
+    })
+
+    it("should modify the repo privacy accordingly if changing to false", async () => {
```

Comment:
[nit]: "should modify the repo privacy accordingly if changing to false" can be "should be able to make repo public"

Reply:
https://github.com/isomerpages/isomercms-backend/pull/807/commits/6154dfd83020b5a74d045fe4d8b7dc94fadf997e